app/views/layouts/application.slim

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
I think, according to [slim website](http://slim-lang.com), that the correct way to do this is `doctype html`.